modules/installation-osp-api-scaling.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-api-scaling_{context}"]
+= Scaling clusters by using Octavia
+
+If your cluster does not use Kuryr, create an Octavia load balancer and then
```

Comment:
this is also valid if using Kuryr, perhaps worth to add a note about it, that with kuryr you can use either this or next section

Reply:
@luis5tb What's an example of when this would be the preferred way of scaling for a Kuryr cluster?

Reply:
@maxwelldb if you want to have several API loadbalancers you will need to follow that approach. Only if you just want 1, then you can reuse the one CNO created for Kuryr